Source/Core/Core/IOS/ES/TitleManagement.cpp

```diff
@@ -440,6 +440,35 @@ IPCCommandResult ES::DeleteTitleContent(const IOCtlVRequest& request)
   return GetDefaultReply(DeleteTitleContent(Memory::Read_U64(request.in_vectors[0].address)));
 }
 
+ReturnCode ES::DeleteContent(u64 title_id, u32 content_id) const
+{
+  if (!CanDeleteTitle(title_id))
+    return ES_EINVAL;
+
+  const auto tmd = IOS::ES::FindInstalledTMD(title_id);
+  if (!tmd.IsValid())
+    return FS_ENOENT;
+
+  IOS::ES::Content content;
+  if (!tmd.FindContentById(content_id, &content))
+    return ES_EINVAL;
+
+  File::Delete(Common::GetTitleContentPath(title_id, Common::FROM_SESSION_ROOT) +
```

Comment:
Whoops, looks like I forgot to check the return value here. It absolutely can fail (and not only with FS_ENOENT; but we can only assume File::Delete returning false means the file doesn't exist.)